athena-federation-integ-test/src/main/java/com/amazonaws/athena/connector/integ/IntegrationTestBase.java

```diff
@@ -258,7 +258,7 @@ protected void cleanUp()
     public Map<String, String> describeTable(String databaseName, String tableName)
             throws RuntimeException
     {
-        String query = String.format("describe %s.%s.%s;", lambdaFunctionName, databaseName, tableName);
+        String query = String.format("describe `%s`.`%s`.`%s`;", lambdaFunctionName, databaseName, tableName);
```

Comment:
same question here

Reply:
Same as above. for DDL you can enclose in back-quote (needed for special characters: e.g. slash '/')